www/js/components/pages/vm/index.jsx

```diff
@@ -544,12 +544,12 @@ var VMPage = React.createClass({
         this.setState({editing: false});
     },
     _handleTagsSave: function (data) {
-        var cnsPropertyName = 'triton.cns.disable';
-        if (Object.prototype.hasOwnProperty.call(data, cnsPropertyName) &&
-            !/^(true|false)$/.test(data[cnsPropertyName])) {
+        var TRITON_CNS_DISABLE_TAG_KEY = 'triton.cns.disable';
+        if (Object.prototype.hasOwnProperty.call(data, TRITON_CNS_DISABLE_TAG_KEY) &&
```

Comment:
don't agree with second part of evaluation. Not empty string will always be true and strict comparison of different types will always be false. 
In other words 'value !== Boolean(value)' will always be true.